moonshot/data/runners-modules/redteaming.py

```diff
@@ -32,12 +43,18 @@ class RedTeaming:
         );
     """
 
+    sql_create_chat_record = """
+        INSERT INTO {} (connection_id,context_strategy,prompt_template,attack_module,
```

Comment:
is this correct? I thought you should put INSERT INTO <table name> (....)

Reply:
yeah this is correct because the table to insert to is an argument that will be passed in

Reply:
okay